src/audio/kpb.c

```diff
@@ -521,7 +521,7 @@ static int kpb_prepare(struct comp_dev *dev)
 	}
 
 	if (!kpb->sel_sink || !kpb->host_sink) {
-		comp_info(dev, "kpb_prepare(): could not find sinks: sel_sink %d host_sink %d",
+		comp_info(dev, "kpb_prepare(): could not find sinks: sel_sink %lu host_sink %lu",
```

Comment:
those are actually pointers. This isn't a bug of this patch, but I don't understand why one would want to print pointers as (decimal) unsigned integers in the first place. So maybe we can use this opportunity and replace those with `%p` and remove type-casts below?